main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"github.com/fogo-sh/gutenbuild/runtime"
+
+	"github.com/BurntSushi/toml"
+
+	"os"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+
+	"io/ioutil"
+)
+
+func main() {
+	zerolog.TimeFieldFormat = zerolog.TimeFormatUnix
+	log.Logger = zerolog.New(os.Stdout)
+
+	log.Print("starting gutenbuild")
+
+	pipeline_content_bytes, err := ioutil.ReadFile("./pipeline.toml")
+	if err != nil {
+		log.Error().Err(err).Msg("error reading pipeline.toml")
```

Comment:
Rather than continuing to decode a file that failed to load, should we exit early here?

```suggestion
		log.Error().Err(err).Msg("error reading pipeline.toml")
		os.Exit(1)
```